### PR TITLE
absentify connector

### PR DESCRIPTION
**Pull request by marc-on-github**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @marc-on-github,
Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks.
Please let us know if you have any questions.
Thank you very much again for working with us.

File: certified-connectors/absentify/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "declined_by_member"
```
Comment (sriyen-msft) [inline certified-connectors/absentify/apiDefinition.swagger.json:420]: @marc-on-github All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/absentify/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+                  "description": "leave_type"
+                }
+              },
+              "description": "details"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {},
```
Comment (sriyen-msft) [inline certified-connectors/absentify/apiDefinition.swagger.json:443]: @marc-on-github Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/absentify/apiDefinition.swagger.json (hunk 1 above)
Comment (marc-on-github) [reply certified-connectors/absentify/apiDefinition.swagger.json]: @sriyen-msft thanks for that feedback. Its fixed

File: certified-connectors/absentify/apiDefinition.swagger.json (hunk 2 above)
Comment (marc-on-github) [reply certified-connectors/absentify/apiDefinition.swagger.json]: @sriyen-msft thanks for that feedback. Its fixed

**Comment by marc-on-github** [comment]
@sriyen-msft I have already fixed everything. Is there anything else I can do?

File: certified-connectors/absentify/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# absentify
+
+absentify is free and specially designed for the Microsoft 365 environment to manage your team's or employee's absences and have an overview of all time off and reason given. You benefit from full integration of paid time off tracking with Microsoft Teams and Microsoft 365, no more boring Excel sheets. absentify is the easiest way to approve vacation time and holidays.
+
```
Comment (v-EgorKozhadei) [inline certified-connectors/absentify/README.md:4]: @marc-on-github please update the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

File: certified-connectors/absentify/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Request created",
+        "description": "This trigger is executed when a new request is created in your company.",
+        "operationId": "trigger_request_created",
```
Comment (v-EgorKozhadei) [inline certified-connectors/absentify/apiDefinition.swagger.json:230]: @marc-on-github operationId properties should [use pascal case](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

File: certified-connectors/absentify/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,496 @@
...
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Request status updated",
+        "description": "This trigger is executed if the status of an existing request is changed.",
+        "operationId": "trigger_request_status_changed",
```
Comment (v-EgorKozhadei) [inline certified-connectors/absentify/apiDefinition.swagger.json:452]: @marc-on-github this one also should be in pascal case, you can refer to [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) for an example.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @marc-on-github,

Thank you for updating the PR! There are a few minor things that need to be fixed. We'll be good to go after that.

Let us know if you have any questions.

File: certified-connectors/absentify/apiDefinition.swagger.json (hunk 4 above)
Comment (marc-on-github) [reply certified-connectors/absentify/apiDefinition.swagger.json]: fixed

File: certified-connectors/absentify/apiDefinition.swagger.json (hunk 5 above)
Comment (marc-on-github) [reply certified-connectors/absentify/apiDefinition.swagger.json]: fixed

File: certified-connectors/absentify/README.md (hunk 3 above)
Comment (marc-on-github) [reply certified-connectors/absentify/README.md:3]: fixed

**Comment by marc-on-github** [comment]
Thanks for your feedback @v-EgorKozhadei I just fixed all things.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @marc-on-github, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**